test/jest/__tests__/stores/appnav.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This has changed from `global` to `appnav` because of feedback from @karynemayer on [this comment](https://github.com/telosnetwork/telos-wallet/pull/266#discussion_r1175847180) in my last PR.

This [issue](https://github.com/telosnetwork/telos-wallet/issues/271) is still open if we need to discuss more about this.